src/charm.py

```diff
@@ -280,9 +283,7 @@ def set_ports(self):
 
     @property
     def _catalogue_item(self) -> CatalogueItem:
-        api_endpoints = {
-            "Alerts": "/api/v2/alerts"
-        }
+        api_endpoints = {"Alerts": "/api/v2/alerts"}
```

Comment:
Side question (non-blocking): Why `tox -e fmt` goes back and forth from one format to another all the time?